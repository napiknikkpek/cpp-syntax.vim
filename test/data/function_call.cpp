foo();
Item::foo();
func(int{3}, std::declval<A>(), a, 3, B{3, a}, B::B0{}, foo(a), C::D::foo(a)))
