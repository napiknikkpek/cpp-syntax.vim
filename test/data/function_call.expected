fff...
tttt..fff...
ffff.zzz.o...ttt..fffffff.t.....x..o..t.o..x...t..tt....fff.x...t..t..fff.x...
